docusaurus.config.js

```diff
@@ -96,51 +96,10 @@ const config = {
             to: "/docs/farming-&-staking/staking/operators/register-operator",
             from: "/docs/farming-&-staking/staking/operators"
           },
-          //Pre-Release Links
-          {
-            to: "/docs/pre-release/participate/contribute",
```

Comment:
One thing I'm not sure about: do we need to set up redirects from /pre-release links to just /docs

Reply:
> One thing I'm not sure about: do we need to set up redirects from /pre-release links to just /docs

I don't think so. The redirects are there to preserve externally linked references. I don't recall us ever having the need to do that with the pre-release docs apart from a handful of Discord messages lost to the sands of time. They can go.